pb_project.yaml

```diff
@@ -1,7 +1,7 @@
 # Project name
 name: base_features
 # Project's yaml schema version
-schema_version: 42
+schema_version: 44
 # WH Connection Profile to use.
 connection: dev_wh
```

Comment:
Can you name the connection as `default`? This is required for the python models to run properly from the UI.